Node-DC-EIS-client/README.md

```diff
@@ -12,9 +12,11 @@ Copyright (c) 2016 Intel Corporation
 
 This client directory contains,
  - runspec.py- A toplevel runspec script; the main script to launch workload.
+     -2 modes: time based run which is the default(mode-1) and request based run (mode-2)
  - node_els-testurls.py - The benchmark driver file which sends actual requests using python's requests module.
  - config.json - The input configuration file.
```

Comment:
Instead of "input configuration file", we should mention "input client configuration file".